core/templates/site/admin/dlqPage.gohtml

```diff
@@ -1,5 +1,5 @@
 {{ template "head" $ }}
-[<a href="/admin">Admin:</a> <a href="/admin/dlq">(This page/Refresh)</a>]<br />
+[<a href="/admin">Admin:</a> <a href="/admin/dlq">(This page/Refresh)</a>]
```

Comment:
The `<br/>` was correct, something about the way `<a>` is being used that is causing the text to break over multiple lines.